.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,28 +17,31 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  
-  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
-  echo "$uri -- $curlResult"  
-  # Temporarily disabled. We're unable to connect to A&A.org from Travis.
-
-  # echo $curlResult |  grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
-  #  || reportBrokenUri "$uri"
+  local output
+  if ! output="$FAILURE_PREFIX$(curl -Lsf -o /dev/null -w "%{http_code}" "$uri")"; then
+    reportBrokenUri "$uri" "$output"
+    FAILURE=1
+  fi
 }
 
 function reportBrokenUri() {
-  FAILURE=1 
   local -r red="\e[38;5;196m"
   local -r blue="\e[38;5;75m"
-  echo -en "Error: ${red}$uri${end} ${blue}"
-  curl -s --head "$uri" | head -n 1 | sed 's|HTTP/[12][.01]?||'
-  echo -en "${end}"
+  local -r error_reason=${2//${FAILURE_PREFIX}000/Connection Failure!}
+  echo -e "Error: ${red}${1}${end} ${blue}${error_reason}${end}"
+}
+
+function checkIsOnIgnoreList() {
+  for ignored_uri in "${IGNORE_LIST[@]}"; do
```

Comment:
Interesting: If `IGNORE_LIST` is empty, this line throws an error on bash 4.3 and prior.
From the [changelog](https://tiswww.case.edu/php/chet/bash/CHANGES):
> a.  Using ${a[@]} or ${a[*]} with an array without any assigned elements when
    the nounset option is enabled no longer throws an unbound variable error.

Which is the reason this build failed: https://travis-ci.com/github/triplea-game/triplea/jobs/342279779